src/style.css

```diff
@@ -0,0 +1,20 @@
+body {
+  font-size: 13px;
+  font-family: Arial;
```

Comment:
indicate subfont if Arial won't be loaded

Reply:
After this definition I have a bug in run test.